.pylintrc

```diff
@@ -0,0 +1,2 @@
+[Format]
+max-line-length=80
```

Comment:
To prevent discussions like this, I'd very much like to remove this limitation completely. 90 is not better than 80, or 120, or whatever. What we have in the source tree seemed to work so far without having to specify a hard limit. As a developer, I'd be extremely pissed if I made a patch against the tools, only to be told by the CI that my lines are too long.